xorca/lib.py

```diff
@@ -46,6 +46,8 @@ def trim_and_squeeze(ds,
         "GLOBAL": {"y": (1, -1), "x": (1, -1)},
         "NEST": {},
     }
+    
+    z_dims = ["z_c", "z_l", "z"]
```

Comment:
Should we add `"deptht", "depthu", "depthv", "depthw"` here?

Also, I'd put this into `orca_names` and use a tuple because it's immutable?

Reply:
I guess we do not need the `depth?`s as long as `rename_dims()` is called before `trim_and_squeeze()`... However, it also won't hurt to include them...